Unix/installbuilder/buildNuget.sh

```diff
@@ -105,12 +105,12 @@ EOF
 # Note, up above, that we are now in the installbuilder/nuget directory
 #
 
-mkdir -p runtimes/linux-x64/native runtimes/osx/native 
```

Comment:
`mkdir -p runtimes/linux-x64/native` need to keep. and only remove `runtimes/osx/native `
otherwise `cp ${build_linux}/libmi.so runtimes/linux-x64/native` will fail.